scripts/mutect2_wdl/mutect2.wdl

```diff
@@ -91,10 +109,23 @@ task SplitIntervals {
     I=${intervals} \
     O=intervalFileDir \
     SCATTER_COUNT=${scatterCount}
+
+    find $PWD/intervalFileDir -iname *.interval_list | sort > interval_lists.txt
```

Comment:
@takutosato In that case you can go ahead and rebase and I will review the rebase.